src/wp-includes/deprecated.php

```diff
@@ -22,27 +22,27 @@
  * @param int $postid Post ID.
  * @return array Post data.
  */
-function get_postdata($postid) {
+function get_postdata( $postid ) {
```

Comment:
Why the PHPCS workflow not detected it? I don't think we needs to fix as it was already deprecated 🤔

Reply:
As the PR is about applying WPCS to the file, I think this fix should also be included.